roles/splunk_common/tasks/add_splunk_license.yml

```diff
@@ -6,6 +6,7 @@
     timeout: 20
   when:
     - splunk.role == "splunk_license_master" or not splunk.license_master_included
+    - splunk.license_uri is match("^(http[s]|file)://.*")
```

Comment:
Just noticing.. shouldn't this be something like `match("^(http|https|file)://.*")` or `match("^(https?|file)://.*")`?

Reply:
Yeah that would be a better rex, I'll amend this